notebooks/python/python-basics.ipynb

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
<p>should and, or, and not have highlighting? <code>and</code>,<code>or</code>,<code>not</code>      </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ESCIP/escip.github.io/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>